ut/mock_s3_motr_reader.h

```diff
@@ -45,6 +45,7 @@ class MockS3MotrReader : public S3MotrReader {
                     std::function<void(void)> on_failed));
   MOCK_METHOD1(get_first_block, size_t(char** data));
   MOCK_METHOD1(get_next_block, size_t(char** data));
+  MOCK_METHOD0(extract_blocks_read, S3BufferSequence());
```

Comment:
MOCK_METHODN is deprecated.

Consider using below:
 MOCK_METHOD(S3BufferSequence, extract_blocks_read, ());

Reply:
Sometimes ago someone (from leads) contradicted against destructing of uniform without reason.

Reply:
"Deprecation of a method" is a good reason to go away with the "uniformity", right ?